lib/less/source-map-output.js

```diff
@@ -104,12 +115,9 @@ module.exports = function (environment) {
         this._sourceMapGenerator = new this._sourceMapGeneratorConstructor({ file: this._outputFilename, sourceRoot: null });
 
         if (this._outputSourceFiles) {
-            for (var filename in this._contentsMap) {
-                if (this._contentsMap.hasOwnProperty(filename)) {
-                    var source = this._contentsMap[filename];
-                    if (this._contentsIgnoredCharsMap[filename]) {
-                        source = source.slice(this._contentsIgnoredCharsMap[filename]);
-                    }
+            for (var filename in this._contentsInfoMap) {
```

Comment:
the bug in the last PR was that this line was still referencing this._contentsMap